dashboard/views/_positions/_ec/_marshal.py

```diff
@@ -190,32 +191,23 @@ def marshal_candidate(request, brother_id):
     return render(request, "brother-view.html", context)
 
 
-@verify_position(['Marshal', 'Vice President', 'President', 'Adviser'])
+@verify_position(['marshal', 'vice-president', 'president', 'adviser'])
 def marshal_candidate_add(request):
     """ Renders the Marshal way of viewing a candidate """
     form = BrotherForm(request.POST or None)
 
     if request.method == 'POST':
         if form.is_valid():
             instance = form.cleaned_data
-            if instance['password'] == instance['password2']:
-                user = User.objects.create_user(instance['case_ID'], instance['case_ID'] + "@case.edu",
-                                                instance['password'])
-                user.last_name = instance['last_name']
-                user.save()
+            user = User.objects.create_user(instance['case_ID'], instance['case_ID'] + "@case.edu",
+                                            instance['password'])
+            user.last_name = instance['last_name']
+            user.save()
```

Comment:
What if the user already exists?

Reply:
tbh idk, I didn't write this part of the code. I imagine an integrity error occurs, but not sure if the form has any validation for that built-in

Reply:
turns out now, time to add some validation to that